### PR TITLE
Adds div plugin to CKEditor and fixes save issues with div els

### PR DESCRIPTION
**Pull request by arellaTV**
Changelog:

- Fixes an issue where inline-styles added to `<div>` elements are stripped out of the body after refreshing by loosening the allowedContent rules in CKEditor.
- Adds the `Div` plugin, which allows an easier method for grouping of elements and applying styles or classes to them.
- Adds the classnames, `.group-wrapper`, `.group-right`, and `.group-left` for a convenient way to float groups right, left, or cleared on both directions.

### CODE REVIEW

File: app/assets/javascripts/outpost/ckeditor/custom-config.js [hunk 1]
```diff
@@ -18,7 +18,7 @@ $(function() {
       "i",
       "b",
       "blockquote",
-      "div[*]",
+      "div[*](*){*}",
```
Comment (nakedsushi) [inline app/assets/javascripts/outpost/ckeditor/custom-config.js]: What does the (*){*} do?

Comment (arellaTV) [reply app/assets/javascripts/outpost/ckeditor/custom-config.js]: @nakedsushi 

I believe the `(*)` denotes that you can apply any class to any `div` element. `{*}` allows for any inline style. Is this maybe too loose? An alternative could be:

`div[*](group-wrapper, group-right, group-left){ float }`

**Comment by arellaTV** [comment]
@nakedsushi 
This is mainly a way to address edge cases in which images that are floated to the right (usually by default because articles float portrait images to the right) overlap with other elements below it. This adds a button so you don't have to add a div and class manually in the source window.

**Comment by arellaTV** [comment]
Got it! Thanks @nakedsushi!

**Pull request merged**